README.md

```diff
@@ -1 +1,109 @@
-# my-enumerables
+## Getting Started
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Enumerables] <a name="about-project"></a>
+
+**[Morse Code]** is a project using Ruby
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Ruby]**
+- **[Enumerables]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Fithamlak**
+
+- GitHub: [@fithamlak](https://github.com/fithamlak)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/fithamlak-fikrie)
+
+👤 **Mirna Ampuero**
+
+- GitHub: [@M1rn4](https://github.com/M1rn4)
+- Twitter: [Mirna_Ampuero](https://twitter.com/_AmpueroMirna)
+- LinkedIn: [Mirna](https://www.linkedin.com/in/mirna-ampuero-caro/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
```

Comment:
I'd advise you to add some features that will improve this codebase or remove this section 🤷🏽 .